lib/mirage_key.ml

```diff
@@ -156,6 +156,12 @@ let warn_error =
   let key = Arg.(opt ~stage:`Configure bool false doc) in
   Key.create "warn_error" key
 
+let gdb =
+  let doc = "Start a gdb server (only available for ukvm)." in
```

Comment:
Hi @mato , actually, what about having the `gdb` module in ukvm start the server by default? We could still have an option to not start it.